### PR TITLE
Update existing Polkadex adapter - Add staking TVL

### PR DESCRIPTION
**Pull request by jaymansfield**
Updating existing Polkadex adapter - Adding staking TVL

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/polkadex exports TVL: 
        
 
 
    polkadex-staking          3.24 M
    staking                   3.24 M
    polkadex                  175.81 k
    
    total                    175.81 k

File: projects/polkadex/index.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+		let staking = getExports("polkadex-staking", ['polkadex']);
+		tvl.polkadex.staking = staking.polkadex.tvl;
+	}
+	return tvl;
+}
+
 module.exports = {
 	timetravel: false,
-	...getExports("polkadex", ['polkadex']),
+	...getPolkadexExports(),
```
Comment (g1nt0ki) [inline projects/polkadex/index.js:14]: change this to `...getExports("polkadex", ['polkadex'], ['staking']),`

**Comment by llamatester** [comment]
The adapter at projects/polkadex exports TVL: 
        
 
 
    polkadex-staking          3.48 M
    staking                   3.48 M
    polkadex                  182.30 k
    
    total                    182.30 k

File: projects/polkadex/index.js (hunk 1 above)
Comment (jaymansfield) [reply projects/polkadex/index.js]: Done

**Pull request merged**